src/pages/Home/index.tsx

```diff
@@ -1,15 +1,25 @@
 import { Container } from 'react-bootstrap';
 import { useEffect, useState } from 'react';
 import { OrderProps, BobaProps } from 'types/common/main';
-import { bobaList, toppingsList, milkList } from 'assets/sampleBobaAPI';
 import BobaContainer from 'components/BobaContainer';
 import SearchBar from 'components/SearchBar';
 import CustomizeModal from 'components/Modal';
+import Loading from 'components/Loading';
 import BobaModalForm from './CustomizeBobaModalBody';
 import Button from 'react-bootstrap/Button';
+import useApi from 'hooks/API';
 
 const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
-  const [filteredBobaList, setFilteredBobaList] = useState<BobaProps[]>(bobaList);
+  const { data, error, loading } = useApi(
+    `${process.env.REACT_APP_BOBA_FETCH}/boba_list` as string,
+  );
+
+  const toppingsListdata = useApi(`${process.env.REACT_APP_BOBA_FETCH}/toppings_option` as string);
+  const toppingsList = toppingsListdata.data;
+  const milkListdata = useApi(`${process.env.REACT_APP_BOBA_FETCH}/milk_option` as string);
```

Comment:
As mentioned before please into two Paramus for host and endpoint for useAPI
Desturcture data and loading instead of setting another variable O(1) space
Use rename during restructure if data is conflicted.

https://wesbos.com/destructuring-renaming

Reply:
updated at c489cd0a2996400916723d7a9510d9d9b0daf1dd